PerformanceTests/JetStream3/in-depth.html

```diff
@@ -655,6 +655,50 @@ <h3>
          Source code: <a href="SunSpider/3d-cube.js">3d-cube.js</a>
         </dd>
 
+        <dt id="bigint-noble-bls12-381">bigint-noble-bls12-381</dt>
```

Comment:
_cc_ @paulmillr: adding your stuff so we could optimize `BigInt` based on real-world usage.

Reply:
@shvaikalesh Sounds awesome. Is there an approximate date for official release of jetstream 3?

Reply:
@paulmillr Per 🍎 policy I can't comment on future releases, even approximately (and I don't even know), yet adding tests for newish JS features is my top priority and I can see light at the end of the tunnel. Also, JavaScriptCore team constantly optimizes newest JS additions without regard to official releases of any benchmark.